agent/src/main/java/io/keploy/agent/KAgent.java

```diff
@@ -82,47 +93,63 @@ public static void premain(String arg, Instrumentation instrumentation) {
                 //for okhttp client interceptor upto version 2.7.5
                 .type(named(okhttp_java))
                 .transform(((builder, typeDescription, classLoader, javaModule, protectionDomain) -> {
-                    logger.debug("inside OkHttpInterceptor_Java transformer");
-                    return builder
-                            .constructor(isDefaultConstructor()).intercept(Advice.to(TypePool.Default.ofSystemLoader().describe("io.keploy.advice.OkHttpAdvice_Java").resolve(), ClassFileLocator.ForClassLoader.ofSystemLoader()));
+                    // if the service (for e.g.: OKHTTP) is not set or set to true, then mock it
+                    if (System.getenv("OKHTTP") == null || (System.getenv("OKHTTP") != null && Boolean.parseBoolean(System.getenv("OKHTTP"))) ) 
+                    {
+                        logger.debug("inside OkHttpInterceptor_Java transformer");
+                        return builder
+                                .constructor(isDefaultConstructor()).intercept(Advice.to(TypePool.Default.ofSystemLoader().describe("io.keploy.advice.OkHttpAdvice_Java").resolve(), ClassFileLocator.ForClassLoader.ofSystemLoader()));
+                    }
+                    // if the env variable is set to false, then return builder; // don't mock this service  don't mock this service 
```

Comment:
Instead of this comment you can add a degub statement  like `skipping x mocking`